package.json

```diff
@@ -15,6 +15,7 @@
     "gatsby-plugin-google-fonts": "0.0.3",
     "gatsby-plugin-less": "^1.0.8",
     "gatsby-plugin-typography": "^1.7.10",
```

Comment:
@nicholasmiller1 `npm` should not be installed locally in a project; it should be a global install on your computer.